inc/saitypes.h

```diff
@@ -1335,6 +1350,9 @@ typedef union _sai_attribute_value_t
 
     /** @validonly meta->attrvaluetype == SAI_ATTR_VALUE_TYPE_LATCH_STATUS */
     sai_latch_status_t latchstatus;
+
+    /** @validonly meta->attrvaluetype == SAI_ATTR_VALUE_TYPE_VLAN_STACKING_VID */
+    sai_vlan_stacking_vid_t vlanstackingvid;
```

Comment:
Do we need a new type ? 
Can we not add more attributes instead for the inner and outer vlan. ?
Not adding a new type to the union helps avoid all the meta changes in SAI and also simplifies meta changes in sonic.

Reply:
If considering this situation, I think change to use individual attributes can achieve the same thing.